core/script.runtime/src/kotlin/script/dependencies/resolvers_deprecated.kt

```diff
@@ -26,7 +26,7 @@ typealias Environment = Map<String, Any?>
 
 interface ScriptDependenciesResolver {
 
-    enum class ReportSeverity { FATAL, ERROR, WARNING, INFO, DEBUG }
+    enum class ReportSeverity { DEBUG, INFO, WARNING, ERROR, FATAL }
```

Comment:
This change is outside the initial scope of the #KT-39502, and making this change will require much wider testing with 3-party projects since it is used in obscured places. On the other hand, this API is in fact obsolete and will be replaced with the  `kotlin.script.experimental.api.ScriptDiagnostic.Severity` soon.
@rnett unless you need this obsolete API to be changed too, I'd drop this one. Otherwise please provide some more details.

Reply:
That seems fine, especially as its obsolete.